pkg/deviceclaimingserver/enddevices/enddevices.go

```diff
@@ -42,7 +41,7 @@ type EndDeviceClaimer interface {
 	// SupportsJoinEUI returns whether the Join Server supports this JoinEUI.
 	SupportsJoinEUI(joinEUI types.EUI64) bool
 	// Claim claims an End Device.
-	Claim(ctx context.Context, req *ttnpb.ClaimEndDeviceRequest) (*ttnpb.EndDeviceIdentifiers, error)
+	Claim(ctx context.Context, joinEUI *types.EUI64, devEUI *types.EUI64, cac string, hNSAddress string) error
```

Comment:
This could be done with an interface but then we need a different package to define the interface to avoid import cycles. This is a simple implementation that works so I've used this.